packages/theming/theme-tokens/src/dark-macos/reactnative/tokens-aliases.json

```diff
@@ -1,262 +1,262 @@
 {
-	"body": {
```

Comment:
I don't think this file should be changed, either. Could you revert the changes in this file as well?

Reply:
Oops, forgot to revert this one

Reply:
Done, thanks for catching!